statmoments/_native.pyx

```diff
@@ -260,12 +267,12 @@ def calc_central_moment_general(raw, ave, n, k, l, i, j):
   com = (-1) ** (k + l) * (1 - k - l) * n * ave_i ** k * ave_j ** l
 
   for p in range(2, k + 1):
-    com += (-1) ** (k + l - p) * binom(k, p) * raw[0,:,p-2,:][i, i]            * ave_j ** l * ave_i ** (k - p)
+    com += (-1) ** (k + l - p) * binom(k, p) * raw[0, :, p - 2, :][i, i] * ave_j ** l * ave_i ** (k - p)
```

Comment:
It's crazy hard to ensure correctness of the formula. I lost hours and days. Could you leave formatted spaces so that `ave` would be upon `ave`

Reply:
I have replaced this with the latest `statmoments/_native.pyx` from [akochepasov:main](https://github.com/akochepasov/statmoments/tree/main)